health-services/plan-service/src/main/java/digit/service/validator/PlanEmployeeAssignmentValidator.java

```diff
@@ -297,7 +292,7 @@ public void validateUpdate(PlanEmployeeAssignmentRequest request) {
         PlanEmployeeAssignment planEmployeeAssignment = request.getPlanEmployeeAssignment();
         String rootTenantId = centralInstanceUtil.getStateLevelTenant(request.getPlanEmployeeAssignment().getTenantId());
         List<PlanConfiguration> planConfigurations = commonUtil.searchPlanConfigId(planEmployeeAssignment.getPlanConfigurationId(), rootTenantId);
-        EmployeeResponse employeeResponse = hrmsUtil.fetchHrmsData(request.getRequestInfo(), planEmployeeAssignment.getEmployeeId(), planEmployeeAssignment.getTenantId());
+        UserDetailResponse userDetailResponse = userUtil.fetchUserDetail(request.getRequestInfo(), planEmployeeAssignment.getEmployeeId(), planEmployeeAssignment.getTenantId());
 
```

Comment:
remove HRMS Util if not being used, remove host and endpoints from app.props, devops